Libplanet/Blockchain/Policies/VolatileStagePolicy.cs

```diff
@@ -273,7 +273,7 @@ public long GetNextTxNonce(BlockChain<T> blockChain, Address address)
             return nonce;
         }
 
-        private bool Exipred(Transaction<T> transaction) =>
+        private bool Expired(Transaction<T> transaction) =>
```

Comment:
Thanks for the help. 😀

However, typos in calls to this method should also be changed for this to work properly. Mind fixing the rest?